.github/workflows/connector-publish.yml

```diff
@@ -0,0 +1,97 @@
+name: Connector Publish workflow
+
+permissions: read-all
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      branch:
+        required: true
```

Comment:
if there is default, is there reason for making required?

Reply:
Removed `required`